src/pages/about.js

```diff
@@ -0,0 +1,101 @@
+import React from "react"
+import { graphql } from "gatsby"
+import { makeStyles } from "@material-ui/core/styles"
+import Img from "gatsby-image"
+import Layout from "../components/layout"
+import SEO from "../components/seo"
+import Typography from '@material-ui/core/Typography'
+import Card from '@material-ui/core/Card'
+import CardMedia from '@material-ui/core/CardMedia';
+
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListItemAvatar from '@material-ui/core/ListItemAvatar';
+import Avatar from '@material-ui/core/Avatar';
+
+const useStyles = makeStyles({
+  card: {
+    display: 'flex',
+    flexWrap: 'wrap'
+  }
+});
+
+function randomColor() {
+  let hex = Math.floor(Math.random() * 0xFFFFFF);
+  let color = "#" + hex.toString(16);
+
+  return color;
+}
+
+const About = ({ data: { enrtfLogo, dynamicUsers } }) => {
+  const classes = useStyles();
+  return (
+    <Layout>
+      <SEO title="About" />
+      {/* Our Mission */}
+      <Typography variant="h2"> Our Mission </Typography>
+      {/* U of M (Not sure if we need this)*/}
+
+      {/* Acknowledge ENTRF funding */}
+      <Card className={classes.card}>
+
+        <Typography>
+          Funding for this project was provided by the Minnesota Environment and Natural Resources
+          Trust Fund as recommended by the Legislative-Citizen Commission on Minnesota Resources (LCCMR)
+          The Trust Fund is a permanent fund constitutionally established by the citizens of Minnesota to
+          assist in the protection, conservation, preservation, and enhancement of the state’s air, water, land,
+          fish, wildlife, and other natural resources.
+          Currently 40% of net Minnesota State Lottery proceeds are dedicated to growing the Trust Fund
+          and ensuring future benefits for Minnesota’s environment and natural resources.
+        </Typography>
+
+        <CardMedia
+          component={Img}
+          fixed={enrtfLogo.childImageSharp.fixed}
+          alt="ENRTF Logo"
+          title="ENRTF Logo"
+        />
+
+      </Card>
+
+      <List component="nav">
+        {dynamicUsers.nodes.map((contributor) => {
+          const displayName = contributor.name? contributor.name : contributor.login;
+          return (
+            <ListItem button component="a" href={contributor.htmlUrl} key={contributor.login}>
+               <ListItemAvatar>
+                <Avatar src={contributor.avatarUrl} style={{backgroundColor: randomColor()}}>
```

Comment:
Instead of doing this random color and using the first letter of the username, we can probably just assume there's always an avatar from GitHub. Even users without a set avatar are assigned a generated one. Setting the background color here seems to cause some styling issues when the actual images are placed in, there is sometimes a slight border around them of this random background color.

For fallbacks when there isn't an avatar somehow, we should use the built-in fallback options provided by MUI: https://material-ui.com/components/avatars/#fallbacks

I think it will automatically use the first letter of the alt text or just use the generic fallback avatar.

Reply:
I was testing those fallbacks methods, but for some reason. I never saw it use first letter of alt text and instead always fall to the generic avatar icon, but yeah I will remove the random color + text combo. I forgot that github assigns a generated avatar as well and having random colors around the borders is definitely undesirable..